modules/examples/src/main/java/com/ibm/cloud/platform_services/global_catalog/v1/GlobalCatalogExamples.java

```diff
@@ -0,0 +1,399 @@
+        }
+
+        try {
+            // begin-get_pricing
+            GetPricingOptions getPricingOptions = new GetPricingOptions.Builder()
+                    .id(catalogEntryId).build();
+
+            Response<PricingGet> response = service.getPricing(getPricingOptions).execute();
+            PricingGet pricingGet = response.getResult();
+
+            System.out.println(pricingGet);
+            // end-get_pricing
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s", e.getStatusCode(),
+                    e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            // begin-get_audit_logs
+            GetAuditLogsOptions getAuditLogsOptions = new GetAuditLogsOptions.Builder()
+                    .id(catalogEntryId)
+                    .offset(0)
+                    .limit(10)
+                    .build();
+
+            Response<AuditSearchResult> response = service.getAuditLogs(getAuditLogsOptions).execute();
+            AuditSearchResult auditSearchResult = response.getResult();
+
+            System.out.println(auditSearchResult);
+            // end-get_audit_logs
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s", e.getStatusCode(),
+                    e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            // begin-upload_artifact
+            String artifactContents = "This is an example artifact associated with a catalog entry.";
+
+            UploadArtifactOptions uploadArtifactOptions = new UploadArtifactOptions.Builder()
+                    .objectId(catalogEntryId)
+                    .artifactId("artifact.txt")
+                    .artifact(new ByteArrayInputStream(artifactContents.getBytes(StandardCharsets.UTF_8)))
+                    .contentType("text/plain")
+                    .build();
+
+            service.uploadArtifact(uploadArtifactOptions).execute();
+            // end-upload_artifact
+        } catch (ServiceResponseException e) {
+            logger.error(String.format("Service returned status code %s: %s\nError details: %s", e.getStatusCode(),
+                    e.getMessage(), e.getDebuggingInfo()), e);
+        }
+
+        try {
+            // begin-get_artifact
+            GetArtifactOptions getArtifactOptions = new GetArtifactOptions.Builder()
+                    .objectId(catalogEntryId)
+                    .artifactId("artifact.txt")
+                    .build();
+
+            Response<InputStream> response = service.getArtifact(getArtifactOptions).execute();
+            InputStream inputStream = response.getResult();
+            if (inputStream != null) {
+                String artifactContents = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
```

Comment:
@pyrooka You might want to use this technique in the case-mgmt examples where you need to read the bytes from the stream returned as a response.